modules/service/src/main/scala/lucuma/odb/graphql/input/GmosImagingInput.scala

```diff
@@ -189,9 +205,10 @@ object GmosImagingInput:
       val Binding: Matcher[South] =
         SouthData.rmap:
           case (filters, exBin, exAmpReadMode, exAmpGain, exRoi, exSpatialOffsets) =>
-            Result(South(
-              filters.flatMap(NonEmptyList.fromList),
```

Comment:
With the `toCreate` in ObservingModeInput removed, this was treating an empty list and a None the same. It now requires either None or a non empty list.